src/App.vue

```diff
@@ -228,12 +230,31 @@ watch(
   }
 )
 
+
+// === Score Chords functions === 
```

Comment:
These are strongly associated with `state`. Makes more sense to move them to that file.

Reply:
Scratch that. You're mixing `scale` and `state`. This is fine here.